changelogs/fragments/6049-dconf-strings.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - dconf - support strings with both single and double quotes (https://github.com/ansible-collections/community.general/pull/6049).
```

```suggestion
  - dconf - support top-level strings with both single and double quotes (https://github.com/ansible-collections/community.general/pull/6049).
```
This change does not apply to strings in arrays etc.